src/views/faq/faq.js

```diff
@@ -36,77 +66,23 @@ class Faq extends React.Component {
       <BackGround>
         <TaskBar goHome={goHome} goRefund={goRefund} goFaq={goFaq} />
         <View className={classes.wrapper}>
-          <View className={classes.tab}>
-            <h1>Boltz FAQ</h1>
-
-            <h2 className={classes.questionTitle}>What is Boltz?</h2>
-            <p>
-              Boltz is an instant and non custodial crypto currency exchange
-              that focuses on the adoption of second layer scaling technologies
-              like the Lightning network and the privacy of its users. We
-              neither use any trackers nor log any data that could identify our
-              users.{' '}
-              <b>Please note that Boltz is currently in its alpha stage</b> so
-              {`don't`} be too reckless when using it. If you experience any
-              issues or have some feedback feel free to{' '}
-              <a href={twitterLink}>contact us</a>.
-            </p>
-
-            <h2 className={classes.questionTitle}>
-              How to open Lightning channels with Boltz?
-            </h2>
-            <p>
-              We appreciate any channels you open with our Lightning nodes and{' '}
-              <a href={twitterLink}>on request</a> we will also open one to you:
-            </p>
-
-            <NodeInfo size={150} name={'Bitcoin LND node'} uri={bitcoinLnd} />
-            <NodeInfo size={150} name={'Litecoin LND node'} uri={litecoinLnd} />
```

Comment:
The rest of the FAQ's is missing. There were two more questions and the URI of the LND node that were removed with your commits.

Reply:
Oh oops, I was sure that the final version is what @peartobear uploaded to zeplin.